src/backend/api/backend.did

```diff
@@ -231,6 +238,7 @@ type CreateProposalReviewRequest = record {
   summary : opt text;
   review_duration_mins : opt nat16;
   build_reproduced : opt bool;
+  vote : opt ProposalVote;
```

Comment:
Is this information submitted manually by reviewers through the form? Or should we fetch it from the NNS based on the reviewers neuron Id?